plugins/linkattacher/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2016 Istituto Italiano di Tecnologia iCub Facility & RBCS Department
+# Authors: Yeshasvi Tirupachuri
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+cmake_minimum_required(VERSION 3.5)
+
+PROJECT(Plugin_LinkAttacher)
+
+include(AddGazeboYarpPluginTarget)
+
+set(linkattacher_sources src/linkattacherserverimpl.cpp src/linkattacher.cc)
+set(linkattacher_headers include/GazeboYarpPlugins/linkattacherserverimpl.h include/gazebo/linkattacher.hh)
+
+set(LIB_COMMON_NAME gazebo_yarp_lib_common)
+if(CMAKE_VERSION VERSION_LESS 3.0.0)
```

Comment:
This condition and related code can be removed now

Reply:
Done